sebastien15/capstoneproject
# Ft retrieve a single article

#### What does this PR do?

>This PR retrieve a single article from the database by finding its id

#### Description of Task to be completed
>added route and method to retrieve a single article by finding its id

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `branch_name`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/single-blog/{id} in post man make sure it is a get method
- you can get all blogs from localhost:3000/all-blogs "get method" and take one id  and use it 



#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/9fXKMyFN
#### Screenshots (if appropriate)
>
![single blog](https://user-images.githubusercontent.com/41580903/93984732-7a71a500-fd84-11ea-8dc8-34105634d210.PNG)
